Scripts/Interfaces/FunctionalityInjection/IRayToNode.cs

```diff
@@ -0,0 +1,18 @@
+﻿#if UNITY_EDITOR
+using System;
+using UnityEngine;
+
+namespace UnityEditor.Experimental.EditorVR
+{
+	/// <summary>
+	/// Provide the ability to request a corresponding node for a ray origin
+	/// </summary>
+	public interface IRayToNode
```

Comment:
If you find you still need this functionality for modules and Workspace, then pull it back into the internal interfaces under Core/Interfaces and remove `public`.